modules/http-api/src/main/java/org/mule/runtime/http/api/client/HttpClient.java

```diff
@@ -166,4 +168,17 @@ default CompletableFuture<WebSocket> openWebSocket(HttpRequest request,
                                                      WebSocketCallback callback) {
     throw new UnsupportedOperationException("WebSockets are only supported in Enterprise Edition");
   }
+
+  /**
+   * Creates a consumer of Server-sent events. The resulting {@link ServerSentEventSource} is not connected automatically.
+   *
+   * @param url         the URL of the server.
+   * @param retryConfig configuration for the retry mechanism.
+   * @return a non-connected instance of {@link ServerSentEventSource}.
+   *
+   * @since 4.10.0
+   */
+  default ServerSentEventSource sseSource(String url, SseRetryConfig retryConfig) {
```

Comment:
`ServerSentEventSource` -> I would rename to `SseSource`

Reply:
Done